storage.rules

```diff
@@ -0,0 +1,8 @@
+rules_version = '2';
+service firebase.storage {
+  match /b/{bucket}/o {
+    match /{allPaths=**} {
+      allow read, write: if request.auth != null;
```

Comment:
If we keep this, I'd change this to `allow read, write: if false` and add a comment that this file only exists to satisfy the emulator.